tests/datatypes/test_bytes.py

```diff
@@ -349,6 +349,29 @@ def test_upper(self):
             print(b'\x46\x55\x43\x4B'.upper())
         """)
 
+    def test_ljust(self):
+        self.assertCodeExecution("""
+            print(b'testMoreThanWidth'.ljust(5))
+            print(b'testEqualWidth'.ljust(14))
+            print(b'testLessThanWidth'.ljust(20))
+            print(b'testMoreWithFill'.ljust(2, b'x'))
+            print(b'testEqualWithFill'.ljust(17, b'x'))
+            print(b'testLessWithFill'.ljust(25, b'x'))
+            print(b'testNegative'.ljust(-20))
+            print(b''.ljust(5))
+            print(b'testNoChangeWidthOne'.ljust(True, b'x'))
+            print(b'testErrorSecondArg'.ljust(True, bytearray(b'x')))
+        """)
+        self.assertCodeExecution("""
+            print(b'test'.ljust('5'))
+        """, exits_early=True)
```

Comment:
`exits_early` is a last resort testing mechanism; it's better to use a try-catch block in the sample code itself, printing something to the console if an exception *isn't* raised (so the test case has something to validate against).

As a side effect, this means you can do the entire test in a single assertCodeExecution block, which is slightly faster - not by a *lot*, but by enough that when you count up all the times we start assertCodeExecution blocks, it adds up.